src/fparser/common/readfortran.py

```diff
@@ -210,6 +210,48 @@ def _is_fix_comment(line, isstrict):
 _IS_CALL_STMT = re.compile(r'call\b', re.I).match
 
 
+def extract_label(line):
+    '''Look for an integer label at the start of 'line' and if there is
+    one then remove it from 'line' and return it in 'label'.
```

Comment:
Strictly speaking this routine does not return 'label'. Please be more explicit or leave the details to the :return: bit of the docstring.

Reply:
Modified.